### DISCUSSION TITLE
How "inkscape:" links are resolved

### DISCUSSION THREAD

**Discussion by buhtz** [jkitchin/scimax#495] [General]
Dear John,
I am not a user of your package and not even Inkscape. But I do serve one of your users in context of my own tool [Hyperorg](https://codeberg.org/buhtz/hyperorg) converting org files into HTML.

I would like to support `[[inkscape:]]` links and would like to know how they are solved? To my understand such links are kind of open-with-inkscape-links, am I right so far?

Is there a full file path in it or some kind of ID I need to resolve?

**Comment by jkitchin** [comment]
There is a path to an svg file in the inkscape link. See https://github.com/jkitchin/scimax/blob/master/scimax-inkscape.el#L94 for example.

There is a minimal export support at https://github.com/jkitchin/scimax/blob/master/scimax-inkscape.el#L165. for html it just makes an img tag.
